testsuite/tests/callback/callback_effects_gc.ml

```diff
@@ -1,6 +1,5 @@
 (* TEST
  ocamlrunparam += ",s=512";
- native;
```

Comment:
I'm not sure why this is part of the PR.

Reply:
Yes, sorry for the noise